admin/templates/base.html

```diff
@@ -0,0 +1,110 @@
+    <!--[if lt IE 9]>
+        <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+        <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+  </head>
+  <body class="skin-blue sidebar-mini">
+    <div class="wrapper">
+
+      <!-- Main Header -->
+      <header class="main-header">
+
+        <!-- Logo -->
+        <a href="{% url 'home' %}" class="logo">
+          <!-- mini logo for sidebar mini 50x50 pixels -->
+          <span class="logo-mini"><b>O</b>SF</span>
+          <!-- logo for regular state and mobile devices -->
+          <span class="logo-lg"><b>OSF</b>Admin</span>
+        </a>
+
+        <!-- Header Navbar -->
+        <nav class="navbar navbar-static-top" role="navigation">
+          <!-- Navbar Right Menu -->
+          <div class="navbar-custom-menu">
+
+            <ul class="nav navbar-nav">
+              <!-- User Account Menu -->
+              <li class="dropdown user user-menu">
+                <a href"#" class="dropdown-toggle" data-toggle="dropdown" aria-expanded="false">
+                  <span class="hidden-xs">Logged in as {{ user.get_username }}</span>
+                </a>
+              </li>
+              <li>
+                <a href="{% url 'auth:logout' %}" class="btn btn-danger btn-flat">Sign out</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+      <!-- Left side column. contains the logo and sidebar -->
+      <aside class="main-sidebar">
+
+        <!-- sidebar: style can be found in sidebar.less -->
+        <section class="sidebar">
+          <!-- Sidebar Menu -->
+          <ul class="sidebar-menu">
+            <li class="header">Menu</li>
+            <li><a href="{% url 'spam:spam' %}"><i class='fa fa-link'></i> <span>OSF Spam</span></a></li>
+            <li><a href="#"><i class='fa fa-link'></i> <span>Pre-reg</span></a></li>
+          </ul><!-- /.sidebar-menu -->
+        </section>
+        <!-- /.sidebar -->
+      </aside>
+
+      <!-- Content Wrapper. Contains page content -->
+      <div class="content-wrapper">
+        <!-- Content Header (Page header) -->
+
+        <!-- Main content -->
+        <section class="content">
+          {% block content %}
+          <h1>
+           Welcome to the OSF Admin Home Page
+          </h1>
+          <!-- Your Page Content Here -->
+          {% endblock %}
+        </section><!-- /.content -->
+      </div><!-- /.content-wrapper -->
+
+      <!-- Main Footer -->
+      <footer class="main-footer">
+        <!-- Default to the left -->
+        <strong>Copyright &copy; 2015 <a href="http://www.osf.io">osf.io</a>.</strong> All rights reserved.
+      </footer>
+
+    </div><!-- ./wrapper -->
+
+    <!-- REQUIRED JS SCRIPTS -->
+
+    <!-- Bootstrap 3.3.2 JS -->
+    {% render_bundle 'vendor' %}
+    <!-- AdminLTE App -->
+    {% render_bundle 'app.min' %}
```

Comment:
I couldn't get this line to not 500. I ended up changing it to `{% render_bundle 'admin-base-page' %}` both here and in login.html.